### PR TITLE
Release 20230928

### PR DESCRIPTION
**Pull request by ElenaFdR**
## Review checklist template

- [x] All new files are linked in indices
- [x] Front matter is present and correct
- [x] Bullets and numbered lists are correct
- [x] Correct Markdown and formatting
- [x] All content reviewed

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ We’re especially interested in learning what you need to implement the [Standa
 
 ## Public organizations
 
-We’re looking for public members to help shape the Foundation for Public Code’s long term future. As a member, you’ll have the chance to shape the Foundation’s processes and governance. Find out more about the [benefits and requirements of becoming a member](https://publiccode.net/membership/) and get in touch with us at <membership@publiccode.net>.
+We’re looking for public partners to help shape the Foundation for Public Code’s long term future. Find out more about the [benefits and requirements of becoming a partner](organization/partnerships.md) and get in touch with us at <partnerships@publiccode.net>.
```
Comment (Ainali) [inline CONTRIBUTING.md:32]: I still can't see this email in our Google groups. Has it been set up in another way?

File: activities/membership-growth/founding-membership-workshop.md
Comment (ElenaFdR) [inline activities/membership-growth/founding-membership-workshop.md:1]: Worth doublechecking that all of the deleted files have an appropriate redirect.

**Review by ericherman — DISMISSED** [review]

File: activities/membership-growth/founding-membership-workshop.md
Comment (Ainali) [reply activities/membership-growth/founding-membership-workshop.md:1]: It looks like these are missing redirects:

- activities/membership-growth/founding-membership-workshop.md
- activities/membership-growth/membership-contract.md
- activities/membership-growth/membership-proposal-template.md

Comment (ElenaFdR) [reply activities/membership-growth/founding-membership-workshop.md:1]: Done!

**Review by Ainali — DISMISSED** [review]

**Review by Ainali — APPROVED** [review]

**Review by ericherman — APPROVED** [review]

**Pull request merged**